data.json

```diff
@@ -8,7 +8,18 @@
         ],
         "type":"musical",
         "year":2005
-    },{
+    },
+    {
+        "title": "27 Dresses"
+        "emojiImgs: "👗📔👰🤵",
```

Comment:
Everything looks pretty good. I noticed that it looks like there is an ending quote missing after `emojiImgs`. Could you update this and resubmit?